cmd/deploy/deploy.go

```diff
@@ -313,10 +325,10 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		}
 	}
 
-	deployOptions.Manifest, err = deployOptions.Manifest.ExpandEnvVars()
-	if err != nil {
-		return err
-	}
+	// deployOptions.Manifest, err = deployOptions.Manifest.ExpandEnvVars()
```

Comment:
???

Reply:
Removing it because it was already expanded after the build step